dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -194,5 +251,13 @@ private static JsonSerializerSettings GetDeserializationSettings()
 
             return settings;
         }
+
+        private static JsonSerializerSettings GetSerializationSettings()
```

Comment:
Overlooked this before, I would just make both the SerializationSettings and DeserializationSettings(irrespective of the number of setting modes), "getters"